### PR TITLE
Moved helper function CreateRuleForBackup from common to backup folder

### PR DESCRIPTION
**Pull request by snigdha-px**
**What this PR does / why we need it**:
https://portworx.atlassian.net/browse/PA-430
**Which issue(s) this PR fixes** (optional)
Closes #
Moved the backup related methods under backup folder 
**Special notes for your reviewer**:

### CODE REVIEW

File: drivers/backup/portworx/portworx.go [hunk 1]
```diff
@@ -1092,6 +1093,120 @@ func (p *portworx) GetBackupUID(ctx context.Context, orgID, backupName string) (
...
+	}
+	ctx, err := backup.GetAdminCtxFromSecret()
+	if err != nil {
+		log.Errorf("Failed to fetch px-central-admin ctx: [%v]", err)
+		return false, ""
+	}
+
+	_, err = p.CreateRule(ctx, RuleCreateReq)
+	if err != nil {
+		log.Errorf("Failed to create backup rules: [%v]", err)
```
Comment (lsrinivas-pure) [inline drivers/backup/portworx/portworx.go:1184]: retrun the error and fail the test with it

File: drivers/backup/portworx/portworx.go [hunk 2]
```diff
@@ -1092,6 +1093,120 @@ func (p *portworx) GetBackupUID(ctx context.Context, orgID, backupName string) (
...
+	}
+	RuleInspectReq := &api.RuleInspectRequest{
+		OrgId: orgID,
+		Name:  ruleName,
+		Uid:   uid,
+	}
+	_, err1 := p.InspectRule(ctx, RuleInspectReq)
+	if err1 != nil {
+		log.Errorf("Failed to validate the created rule with Error: [%v]", err)
+		return false, uid
```
Comment (lsrinivas-pure) [inline drivers/backup/portworx/portworx.go:1206]: same here

File: tests/backup/backup_test.go [hunk 3]
```diff
@@ -259,10 +269,10 @@ var _ = Describe("{BasicBackupCreation}", func() {
 
 	})
 	JustAfterEach(func() {
-		policy_list := []string{"interval", "daily", "weekly", "monthly"}
+		//policy_list := []string{"interval", "daily", "weekly", "monthly"}
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:272]: delete if not required

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]

File: drivers/backup/portworx/portworx.go (hunk 1 above)
Comment (snigdha-px) [reply drivers/backup/portworx/portworx.go:1184]: done!

File: drivers/backup/portworx/portworx.go (hunk 2 above)
Comment (snigdha-px) [reply drivers/backup/portworx/portworx.go:1206]: done

File: tests/backup/backup_test.go (hunk 3 above)
Comment (snigdha-px) [reply tests/backup/backup_test.go:272]: done

**Pull request closed without merging**